python/ray/setup-dev.py

```diff
@@ -75,6 +75,7 @@ def do_link(package, force=False, local_path=None):
     do_link("util", force=args.yes)
     do_link("serve", force=args.yes)
     do_link("_private", force=args.yes)
+    do_link("node.py", force=args.yes)
```

Comment:
Is this intentional?

Reply:
Yes.
I've just started adding things to setup-dev when they cause me issues. 
(added _private in another PR)

I could remove this and add a PR later to link all of the python files in the root directory.

Reply:
No worries, it seems fine to include it

Reply:
okidoke